frameworks/helloworld/tests/test_host_volumes.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import retrying
+import pytest
+
+import sdk_cmd
+import sdk_install
+from tests import config
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+        options = {"service": {"yaml": "host-volume"}}
+
+        sdk_install.install(config.PACKAGE_NAME, config.SERVICE_NAME, 2, additional_options=options)
+
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.hostvolume
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_check_host_volume_mounts():
+    """world has no docker image defined, hello does. Check to make sure the host volume is mounted in the container"""
+    assert "host-volume-etc" in search_for_host_volume(
```

Comment:
As a sanity check, how about testing that we're able to access a file in the `/etc` volume?

Like `host-volume-etc/passwd` should contain a line that starts with `root:`